index.php

```diff
@@ -0,0 +1,32 @@
+<?php
+require_once 'src/ViewHelpers/CarsViewHelper.php';
+require_once 'src/Models/CarsModel.php';
+require_once 'src/Entities/Car.php';
+
+$db = new PDO('mysql:host=db; dbname=Cars', 'root', 'password');
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+$cars_model = new CarsModel($db);
+$allCars = $cars_model->getAllCars();
```

Comment:
You have some inconsistency here with snake and camel case, pick one and stick with it!